.gitignore

```diff
@@ -0,0 +1,7 @@
+f90wrap_pyhmcode/
+include/
+lib/
+build/
+
+.vscode
+.DS_Store
```

Comment:
Is it necessary to contain this .gitignore file with HMcode? I currently have my own .gitignore, which ignores itself. Could that be used here?

Reply:
I think shipping a .gitignore with common files and folders that should be ignored makes sense. Else everyone has bunch of untracked files they need to deal with.